src/Cygnus/DrushExport/AbstractExport.php

```diff
@@ -626,6 +639,7 @@ protected function importWebsiteSectionNodes()
 
         $formatted = [];
         foreach ($nodes as $node) {
+            // @jpdev - 20190102 - just $n - broken? We don't import sections so didn't confirm - fix?
```

Comment:
yes, there likely isn't a valid translation for the B4 `description field`. Remove this line